src/Compilers/Core/Portable/Collections/OrderPreservingMultiDictionary.cs

```diff
@@ -99,13 +103,13 @@ public void Add(K k, V v)
             else
             {
                 this.EnsureDictionary();
-                _dictionary[k] = new ValueSet(v);
+                _dictionary![k] = new ValueSet(v);
```

Comment:
>new ValueSet(v) [](start = 34, length = 15)

I'm actually very surprised this isn't giving a warning, given that the constructor has a `NotNull` attribute on it. Filed [https://github.com/dotnet/roslyn/issues/40141](https://github.com/dotnet/roslyn/issues/40141) to track, this should be warning.